guides/introduction/installation.md

```diff
@@ -90,6 +90,8 @@ When we work with Ecto schemas in these guides, we will use PostgreSQL and the P
 
 Postgrex is a direct Phoenix dependency, and it will be automatically installed along with the rest of our dependencies as we start building our app.
 
+Mac users can optionally install [Postgres.app](https://postgresapp.com) to automatically start PostgreSQL.
```

Comment:
Isn't this redundant if a link to the installation guides (which also mention Postgres.app) is given just a few lines above? Yes, installing alone won't be enough, Postgres needs to be started as well. This could be explicitly stated.

Maybe improving the wording of the paragraph above is the better solution here. For the sake of keeping the guide short and as platform-agnostic as possible. :)

Reply:
You're right, it's not necessary to add OS-specific instructions in order to add clarity. I'll make a different edit in another PR.